GetF!t/Modules/Exercises/Timer/TimerView.swift

```diff
@@ -1,9 +1,10 @@
 import SwiftUI
 
 struct TimerView: View {
-    @StateObject var timerViewModel = TimerViewModel(exerciseName: LocalizedStringProvider.ExercisesNames.squat)
+    @ObservedObject var timerViewModel: TimerViewModel
     @Environment(\.presentationMode) var presentationMode
-
+    @EnvironmentObject var selectedTabManager: SelectedTabManager
```

Comment:
😄  😄  no.. zatiaľ to tu nechaj, ale potom to asi spravíme inak. A prosím nechaj ten TimerViewModel ako  @StateObject v tomto prípade :) a inicializuj ho priamo tu vo view ako predtým

Reply:
to jsem chtel ale se state objectem mi to padalo hned jak jsem zmackl Start exercise

Reply:
mne to funguje takže si asi len ten object neinicializoval alebo niečo. Skús to ešte raz :)

Reply:
je mi to lito ale nevim jak nefunguje to

Reply:
done